### PR TITLE
osa - pin goimports to commit 04b5d21e in azure base image

### PR DESCRIPTION
**Pull request by Makdaam**
Pinning goimports to a specific commit as requested in pull:
https://github.com/openshift/openshift-azure/pull/862#issuecomment-445252070

### CODE REVIEW

File: projects/azure/base-images/test-base.yaml [hunk 1]
```diff
@@ -61,11 +61,18 @@ items:
...
         RUN go get github.com/go-bindata/go-bindata && \
             go get golang.org/x/tools/cmd/goyacc && \
-            go get golang.org/x/tools/cmd/goimports && \
             go get github.com/golang/mock/gomock && \
             go install github.com/golang/mock/mockgen && \
-            mv $GOPATH/bin/* /usr/bin/ && \
-            rm -rf $GOPATH/* $GOPATH/.cache
+            mv $GOPATH/bin/* /usr/bin/
+
+        #pin goimports to 04b5d21e commit
```
Comment (mjudeikis) [inline projects/azure/base-images/test-base.yaml]: Can you add a comment to the discussion to why.
@kargakis we should migrate to go modules at some point, and replace all these?

**Comment by kargakis** [comment]
@Makdaam can you also try triggering the build in the CI cluster? Builds are not running automatically once they are updated.

**Comment by Makdaam** [comment]
@kargakis new image built successfully.

**Pull request merged**